sql/core/src/test/java/test/org/apache/spark/sql/JavaDatasetSuite.java

```diff
@@ -309,6 +309,7 @@ public void testSetOperation() {
   }
 
   @SafeVarargs
+  @SuppressWarnings("varargs")
```

Comment:
That should be OK if it suppresses the warning, but I'm not sure why `@SafeVarargs`, which is the more appropriate annotation, doesn't already do so.